config/locales/en/user/registrations.yml

```diff
@@ -7,6 +7,8 @@ en:
           Already registered? If you have an existing Coursemology account, you can %{sign_in}.
         password_hint: "#{length} characters minimum"
         sign_up: :'user.registrations.new.header'
+        used: 'The invitation code is used by another account, please login that account instead.'
+        used_with_email: 'The invitation code is used by %{email}, please login using that email instead.'
```

Comment:
The provided invitation code has been claimed by %{email}, please use that account to access the course instead.